data-structures-stacks/is-empty.js

```diff
@@ -0,0 +1,7 @@
+export default function isEmpty(stack) {
+  if (stack.peek() === undefined) {
```

Comment:
Don't use `if (x) return true; else return false;`. It's ugly and amateurish. Instead just `return x;`. If you need to return a boolean and `x` is not a boolean, use `return !!x;`.